src/components/ReleaseService/ReleasePlan/TriggerRelease/AddIssueSection/BugFormContent.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import {
+  Button,
+  ButtonType,
+  ButtonVariant,
+  Form,
+  Stack,
+  StackItem,
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+import { useFormikContext } from 'formik';
+import { isEmpty } from 'lodash-es';
+import { InputField, TextAreaField } from '../../../../../shared';
+import StatusDropdown from './StatusDropdown';
+import UploadDate from './UploadDate';
+
+type BugFormValues = {
+  issueKey: string;
+  url: string;
+  uploadDate: string;
+  status: string;
+};
+
+type BugFormContentProps = {
+  modalToggle: () => void;
+};
+const BugFormContent: React.FC<BugFormContentProps> = ({ modalToggle }) => {
+  const { handleSubmit, isSubmitting, errors, dirty } = useFormikContext<BugFormValues>();
+
+  return (
+    <Form>
+      <Stack hasGutter>
+        <StackItem>
+          <TextContent>
+            <Text component={TextVariants.p}>
+              Provide information about a Bug that has already been resolved.
+            </Text>
+          </TextContent>
+        </StackItem>
+        <StackItem>
+          <InputField data-test="bug-issue-key" label="Bug issue key" name="key" required />
+        </StackItem>
+        <StackItem>
+          <InputField data-test="bug-url" label="URL" name="url" required />
+        </StackItem>
+        <StackItem>
```

Comment:
Date picker field is missing "Last updated" title.

<img width="404" alt="image" src="https://github.com/openshift/hac-dev/assets/9964343/24b5c1d5-47c0-4f77-b722-7bb2d09e333a">